kasa/aestransport.py

```diff
@@ -77,7 +82,7 @@ def __init__(
         self._http_client: httpx.AsyncClient = httpx.AsyncClient()
         self._login_token = None
 
-        _LOGGER.debug("Created AES object for %s", self.host)
+        _LOGGER.debug("Created AES object for %s", self._host)
```

Comment:
```suggestion
        _LOGGER.debug("Created AES transport for %s", self._host)
```
The "Created AES|KLAP objects" are some left-overs, so we could change them now here as the line is modified anyway.